test/nimble_template/addons/variants/health_plug_test.exs

```diff
@@ -0,0 +1,82 @@
+defmodule NimbleTemplate.Addons.Phoenix.HealthPlugTest do
+  use NimbleTemplate.AddonCase, async: false
+
+  describe "#apply/2" do
+    @describetag required_addons: [:ExCoveralls, :"Phoenix.MixRelease"]
+
+    test "copies the health plug file", %{
+      project: project,
+      test_project_path: test_project_path
+    } do
+      in_test_project(test_project_path, fn ->
+        PhoenixAddons.HealthPlug.apply(project)
+
+        assert_file("lib/nimble_template_web/plugs/health_plug.ex")
+      end)
+    end
+
+    test "copies the health plug test files", %{
+      project: project,
+      test_project_path: test_project_path
+    } do
+      in_test_project(test_project_path, fn ->
+        PhoenixAddons.HealthPlug.apply(project)
+
+        assert_file("test/nimble_template_web/plugs/health_plug_test.exs")
+        assert_file("test/nimble_template_web/requests/_health/liveness_request_test.exs")
+        assert_file("test/nimble_template_web/requests/_health/rediness_request_test.exs")
+      end)
+    end
+
+    test "adds health_path configuration in config", %{
```

Comment:
Sometimes we're referring `health_path` or `health path`. Should we update it to be consistent 💡 
```suggestion
    test "adds health path configuration in config", %{
```

Reply:
adjusted it in https://github.com/nimblehq/elixir-templates/pull/159/commits/06a24ca05d89961279d29d39e0542e117dafe7c4 @rosle